python/test/unit/language/test_compile_only.py

```diff
@@ -54,7 +54,7 @@ def simple_dot(a_base, b_base, out):
     assert re.search(pattern, str(ttgir)), "The TTGIR does not match the expected pattern."
 
     ptx = k.asm["ptx"]
-    pattern = (r"mov\.u32 	%r(?P<G>\d+), global_smem;"
+    pattern = (r"mov\.b32 	%r(?P<G>\d+), global_smem;"
```

Comment:
Not sure about the significance here?

Reply:
cc @ThomasRaoux

Reply:
should be equivalent. Thanks for pointing this out